.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: python
 python:
   - '2.7'
-  - '3.5'
```

Comment:
still active

Reply:
right, but I don't want to test on every single version just for the sake of it, I'm keeping 2.7 a while longer, and I will keep testing the last 2-3 3.x releases only.